routing/dht/routing.go

```diff
@@ -122,10 +122,11 @@ func (dht *IpfsDHT) GetValue(ctx context.Context, key u.Key) ([]byte, error) {
 // Provide makes this node announce that it can provide a value for the given key
 func (dht *IpfsDHT) Provide(ctx context.Context, key u.Key) error {
 	log := dht.log().Prefix("Provide(%s)", key)
+
 	log.Debugf("start", key)
```

Comment:
can we remove these two debug statements? they're so low signal...

log.Debugf("start", key)
defer log.Debugf("end", key)